cli/tests/unit/read_dir_test.ts

```diff
@@ -19,6 +19,11 @@ unitTest({ perms: { read: true } }, function readDirSyncSuccess(): void {
   assertSameContent(files);
 });
 
+unitTest({ perms: { read: true } }, function readDirSyncWithUrl(): void {
+  const files = [...Deno.readDirSync(new URL("file:./cli/tests"))];
```

Comment:
Although annoying, I don't think you can create cwd-relative file URLs like that.

Other platforms don't agree but none of them produces this outcome.

Chrome:
```
> new URL("file:./cli/tests")
URL {href: "file://./cli/tests", origin: "file://", protocol: "file:", username: "", password: "", …}
```

Node:
```
D:\deno>node
Welcome to Node.js v14.2.0.
Type ".help" for more information.
> new URL("file:./cli/tests")
URL {
  href: 'file:///cli/tests',
  origin: 'null',
  protocol: 'file:',
  username: '',
  password: '',
  host: '',
  hostname: '',
  port: '',
  pathname: '/cli/tests',
  search: '',
  searchParams: URLSearchParams {},
  hash: ''
}
```

Firefox:
```
> new URL("file:./cli/tests")
URL { href: "file:///cli/tests", origin: "null", protocol: "file:", username: "", password: "", host: "", hostname: "", port: "", pathname: "/cli/tests", search: "" }
```

Reply:
@piscisaureus You're right, I looked back at RFC 8089 (which I know isn't quite accepted yet but it's the most complete description of the syntax I can find) and I have no idea where I got the syntax I'm using there. I'll revert back to absolute paths sometime this weekend.

Speaking of, based on whatwg+RFC 8089 it seems like the only way to do a relative URL of any kind is "scheme-less" but no `URL` object on any browser actually accepts it. Not sure what that's about, but I guess it doesn't affect this PR much anyway.